server/src/routes/schedule.js

```diff
@@ -264,109 +264,123 @@ router.put('/interviewer/delete/:id', (req, res) => {
 // find all the possible meeting times, given the following constraints/information:
 // attendess, timeConstraints, meetingDuration, locationConstraints
 router.post('/meeting', notAuthMiddleware, async (req, res) => {
-  const { candidate, meetingDuration, required, optional } = req.body;
+  const {
+    candidate, meetingDuration, required, optional,
+  } = req.body;
   const sql = "SELECT * FROM Candidate c INNER JOIN CandidateAvailability a ON c.id = a.candidateID WHERE c.email = ? AND c.status = 'A' ORDER BY a.id DESC";
-  const sqlcmd = connection.format(sql, [candidate]);
+  const getCandidateAvailabilityCmd = connection.format(sql, [candidate]);
```

Comment:
Just curious, but why did you relabel this?

Reply:
Because there are multiple sql commands in here haha Eslint is complaining and also it's clearer if we're refactoring this endpoint :)